### PR TITLE
chore(docs): compilation for tooltip and popover

### PR DESCRIPTION
**Pull request by icfantv**
* update doc for tooltip and popover to indicate the HTML returned by the
  `uib-*-html` attribute will not be compiled.
* address complaints in #4923.

### CODE REVIEW

File: src/popover/docs/readme.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ There are three versions of the popover: `uib-popover` and `uib-popover-template
   Takes text only and will escape any HTML provided for the popover body.
   
 * `uib-popover-html`
-  Takes an expression that evaluates to an html string. *The user is responsible for ensuring the content is safe to put into the DOM!*
+  Takes an expression that evaluates to an html string.  Note that this HTML is not compiled.  If compilation is required, please use the `uib-popover-template` attribute option instead. *The user is responsible for ensuring the content is safe to put into the DOM!*
```
Comment (wesleycho) [inline src/popover/docs/readme.md]: Super minor, but should only use 1 space after a period - browsers eat up any extra spacing.

File: src/tooltip/docs/readme.md [hunk 2]
```diff
@@ -6,10 +6,10 @@ There are three versions of the tooltip: `uib-tooltip`, `uib-tooltip-template`,
 
 * `uib-tooltip` -
   Takes text only and will escape any HTML provided.
+* `uib-tooltip-html` -
+  Takes an expression that evaluates to an html string.  Note that this HTML is not compiled.  If compilation is required, please use the `uib-tooltip-template` attribute option instead. *The user is responsible for ensuring the content is safe to put into the DOM!*
```
Comment (wesleycho) [inline src/tooltip/docs/readme.md]: Ditto

File: src/popover/docs/readme.md (hunk 1 above)
Comment (icfantv) [inline src/popover/docs/readme.md]: habit...will fix.

**Comment by icfantv** [comment]
screwed up rebase...

**Pull request closed without merging**